### ISSUE TITLE
how to train？

### ISSUE THREAD

**Issue by liu09114** [FengyangZhang/caffe_roomnet#2]
@FengyangZhang 
Thanks for your released code.
I want to train a model for keypoints prediction, but I can't get the type of input data.
Will you offer the code for preparing data? Or just release the type of 'train.txt' as in the train.prototxt?

**Comment by FengyangZhang** [comment]
Hi Liu,
It's a comma-separated text file, I used some simple script to produce it and I don't have the script anymore. However,  I think you can find out the format in src/caffe/layers/data_heatmap.cpp when the code reads the input data into memory.

**Comment by liu09114** [comment]
Yes, I have read the layer code data_heatmap.cpp. Since now I just want to have a quick test on the keypoint detection. Could you provide your 'train.txt' to me? My email is liu09114@gmail.com .
Best wishes.

**Comment by FengyangZhang** [comment]
I'm sorry but I don't have that now..

**Comment by zhangqiudan** [comment]
@FengyangZhang 
Thanks for your code.
I want to know the form of 'train.txt', can you give me a simple example about the content of the 'train.txt' file? just like this img_name, typestr represents the type of image, but i am not sure the labels content.
